modules/rosa-upgrading-automatic-ocm.adoc

```diff
@@ -0,0 +1,23 @@
+:_module-type: PROCEDURE
+//Specify the system-module-type as either "CONCEPT, PROCEDURE, or REFERENCE"
+
+// Module included in the following assemblies:
+//
+// * assemblies/rosa-upgrading.adoc
+
+[id="rosa-scheduling-upgrade_{context}"]
+= Scheduling automatic upgrades
+
+[role="_abstract"]
+There are three methods that can be used to upgrade {product-title} clusters. This procedure describes scheduling automatic upgrades.
+
+.Procedure
+
+. Log in to the link:https://cloud.redhat.com/openshift[OCM console].
+. Select a cluster to upgrade.
+. Click the *Update settings* tab.
+. In the _Update Strategy_ pane, click *Automatic*.
+. Scheduling options display. Select the day of the week and the time for upgrades to occur.
```

Comment:
s/options display/options are displayed.
**! display** _v_
Use only as a transitive verb. For example, write “the message is displayed” or “the system displays a
message,” not “the message displays.”

Reply:
Done.